### PR TITLE
Week4

### PR DESCRIPTION
**Pull request by oledu**
終於完成了！

### CODE REVIEW

File: homeworks/week4/hw3/index.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        <div class="shot"></div>
+        <div class="info"></div>
+
+      </div>
+      <div class="row__window">
+        <img>
+        <div class="shot"></div>
+        <div class="info"></div>
+      </div>
+    </div>
```
Comment (aszx87410) [inline homeworks/week4/hw3/index.html:144]: 讓你思考一個問題：如果我今天說要顯示 100 個怎麼辦？

File: homeworks/week4/hw5.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  JSON 是資料儲存的格式，方便處理也容易閱讀。格式類似以下：
+  {
+  "firstName": "Peter",
+  "lastName": "Xu",
+  "sex": "male",
+  "age": 25,
+  }
+
+7. JSONP 是什麼？
+  JASONP 是一種利用 <script> 標籤的特性，讓網頁可以繞過 Same Origin Policy 來得到 response 的資料。
```
Comment (aszx87410) [inline homeworks/week4/hw5.md:41]: 多一個 A

**Comment by aszx87410** [comment]
https://lidemy.github.io/mentor-program-2nd-oledu/homeworks/week4/hw1/index.html
https://lidemy.github.io/mentor-program-2nd-oledu/homeworks/week4/hw2/index.html
https://lidemy.github.io/mentor-program-2nd-oledu/homeworks/week4/hw3/index.html

簡答題答的不錯，作業大致上都 ok，恭喜完成作業四 🎉

**Comment by oledu** [comment]
感謝批改
修訂已經新開pr了！

**Pull request merged**